x/staking/handler_test.go

```diff
@@ -95,9 +95,9 @@ func TestValidatorByPowerIndex(t *testing.T) {
 
 	ctx = ctx.WithBlockTime(finishTime)
 	header := abci.Header{Entropy: testBlockEntropy()}
-	BeginBlocker(ctx, abci.RequestBeginBlock{Header: header}, &keeper)
-	EndBlocker(ctx, &keeper)
-	EndBlocker(ctx, &keeper)
+	BeginBlocker(ctx, abci.RequestBeginBlock{Header: header}, keeper)
+	EndBlocker(ctx, keeper)
+	EndBlocker(ctx, keeper)
```

Comment:
Double end blocker - intentional?

Reply:
It was in the original test, but the test seems to run fine without it so it was probably just a typo.